modules/gdscript/gdscript_byte_codegen.cpp

```diff
@@ -162,6 +162,7 @@ void GDScriptByteCodeGenerator::end_parameters() {
 void GDScriptByteCodeGenerator::write_start(GDScript *p_script, const StringName &p_function_name, bool p_static, Variant p_rpc_config, const GDScriptDataType &p_return_type) {
 	function = memnew(GDScriptFunction);
 	debug_stack = GDScriptLanguage::get_singleton()->should_track_locals();
+	enable_newline = GDScriptLanguage::get_singleton()->should_track_call_stack();
```

Comment:
I'd rename `enable_newline` and `debug_stack` to `track_call_stack` and `track_locals` for consistency, like in `GDScriptLanguage` (or we could call these methods directly in the code generator). Especially given the potential confusion that `should_track_call_stack()` corresponds to `enable_newline` rather than `debug_stack`.